### ISSUE TITLE
Export from iCloud Photos Library?

### ISSUE THREAD

**Issue by haoxi911** [tymmej/ExportPhotosLibrary#4]
Hello Jakub,

First of all, thank you for sharing your approach with us, it is pretty clear solution and easy to use. My only concern is about `iCloud Photos Library` support, as you may know, when this option is turned on in Photos app, it will automatically upload the original copies into iCloud so they will not be existed on local disk.

Do you have any insights on support this? Will there be kind of links to iCloud server in the database so that we can download photos via HTTP?

Thanks!
Kevin

**Comment by tymmej** [comment]
Do you have "Optimize storage" turned on?

**Comment by haoxi911** [comment]
@tymmej Yes, we turned on that option and the customer is in low disk space, so local version of her photos have been removed.
